testsuite/features/secondary/srv_timezone_feature

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+#  1) create a user and assing him a timezone different than the server's timezone
+#  2) test that the popups in some scheduling actions appear in users prefered timezone 
+#  3) some scheduler tests based on previous bugs are unavoidable
+
+#@sle_minion
```

Comment:
Be caution with the commented tag, in case we really want to use a tag here.

Reply:
yes , this is mostly an open PR , I will proceed and replace all my minion names with sle_minion and use @sle_minion on top